src/DssSpell.t.sol

```diff
@@ -25,177 +25,10 @@ interface LPTokenLike {
     function token1() external view returns (address);
 }
 
-// Specific for this spell
-interface RwaInputConduitLike {
-    function push() external;
+interface DSValueLike {
+    function read() external view returns (bytes32);
```

Comment:
FWIW, you've got access to [DSValueAbstract](https://github.com/makerdao/dss-interfaces/blob/master/src/dapp/DSValueAbstract.sol) in the test via the `Interfaces.sol` import.